buildSrc/src/main/groovy/com/github/okbuilds/core/model/AndroidTarget.groovy

```diff
@@ -65,12 +66,12 @@ abstract class AndroidTarget extends JavaLibTarget {
 
     @Override
     Scope getTest() {
-        return new Scope(project,
-                ["testCompile", "${buildType}TestCompile", "${flavor}TestCompile", "${name}TestCompile"] as Set,
-                baseVariant.sourceSets.collect { SourceProvider provider ->
-                    provider.javaDirectories
-                }.flatten() as Set<File>,
-                null,
+        return new Scope(
+                project,
+                ["compile", "${buildType}Compile", "${flavor}Compile", "${name}Compile",
+                 "testCompile", "${buildType}TestCompile", "${flavor}TestCompile", "${name}TestCompile"] as Set,
+                project.files("src/test/java") as Set<File>,
```

Comment:
discussed offline, testVariant is null for library variants so we will keep as is.